apps/react/src/api/services/studioService.ts

```diff
@@ -0,0 +1,29 @@
+import { AnimeStudio } from '@js-camp/core/models/studio.model';
+import { StudioDto } from '@js-camp/core/dtos/studio.dto';
+import { StudioMapper } from '@js-camp/core/mappers/studio.mapper';
+import { StudioQueryParameters } from '@js-camp/core/models/studio-query-parameters.model';
+import { StudioQueryParametersMapper } from '@js-camp/core/mappers/studio-query-parameters.mapper';
+import { ListCursorPaginationDto } from '@js-camp/core/dtos/list-cursor-pagination.dto';
+import { ListCursorPaginationMapper } from '@js-camp/core/mappers/list-cursor-pagination.mapper';
+import { ListCursorPagination } from '@js-camp/core/models/list-cursor-pagination.model';
+
+import { http } from '..';
+
+import { AppUrlConfigService } from './appUrlConfigService';
+
+const url = AppUrlConfigService.paths.studioList;
+
+export namespace StudiosService {
+
+	/**
+	 * Fetches list of studios.
+	 * @param queryParams Studio query parameters.
+	 * @returns
+	 */
+	export async function fetchStudios(queryParams: StudioQueryParameters): Promise<ListCursorPagination<AnimeStudio>> {
+		const { data } = await http.get<ListCursorPaginationDto<StudioDto>>(url, {
```

Comment:
Let's move 'url' variable closer to place where it's used:
```suggestion
export namespace StudiosService {

	/**
	 * Fetches list of studios.
	 * @param queryParams Studio query parameters.
	 */
	export async function fetchStudios(queryParams: StudioQueryParameters): Promise<ListCursorPagination<AnimeStudio>> {
		const url = AppUrlConfigService.paths.studioList;
		const { data } = await http.get<ListCursorPaginationDto<StudioDto>>(url, {
```